src/app/components/checkout/checkout.component.ts

```diff
@@ -49,7 +49,9 @@ export class CheckoutComponent implements OnInit {
     this.carts$.subscribe((items) => {
       let total = 0;
       for (const item of items) {
-        total += item.price * item.orderedUnit;
+        if(item.paymentOption === 'online'){
```

Comment:
Instead of using magic string , use a constant for online , so it can be maintained in one place, you can add it in the shared folder

Reply:
fixed